src/index.html

```diff
@@ -1,13 +1,440 @@
+                <a href="#contactUs" class="nav__link nav__link--last">Contact us</a>
+              </li>
+            </ul>
+          </div>
+          <a href="#menu" class="header__menu-opener"></a>
+        </nav>
+
+
+        <div class="grid grid--desktop">
+          <p class="
+                  header__about
+                  grid__item
+                  grid__item--onTablet--1-4
+                  grid__item--onDesKtop--1-2"
+          >
+            We bake your fantasies — croissants, cakes, cookies, and cupcakes.
+          </p>
+        </div>
+        <h1 class="header__title">Creative bakery</h1>
+        <img
+          src="./images/Cookie.png"
+          alt="main-image"
+          class="header__image"
+        >
+
+        <div class="grid grid--desktop">
+          <div class="header__button
+                      grid__item
+                      grid__item--onTablet--3-6
+                      grid__item--onDesKtop--1-2">
+            <a href="#aboutUs" class="button">
+              Learn more
+            </a>
+          </div>
+
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <nav class="menu__nav">
+          <a href="#" class="logo">
+            <img
+              src="./images/Logo.svg"
+              alt="Bakery"
+              class="logo__image"
+            >
+          </a>
+          <a href="#" class="menu__close"></a>
+        </nav>
+        <div class="menu__container">
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#products" class="menu__link">Products</a>
+            </li>
+            <li class="menu__item">
+              <a href="#aboutUs" class="menu__link">About Us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#Blog" class="menu__link">Blog</a>
+            </li>
+            <li class="menu__item menu__item--last">
+              <a href="#contactUs" class="menu__link nav__link--last">Contact us</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+   </div>
+
+    <main class="page__main">
+
+      <section class="page__section" id="products">
+        <div class="container">
+          <h2 class="page__section-title">
+            What we bake
+          </h2>
+          <p class="page__section-subTitle">
+            We can customize our cookies right for your needs. Just imagine it
+          </p>
+
+          <div class="container__card">
+            <div class="grid grid--desktop">
+              <div class="page__card card
+                          grid__item
+                          grid__item--onTablet--1-4
+                          grid__item--onDesKtop--1-3"
+              >
+                <img
+                    src="./images/photo-product-01.png"
+                    alt="photo-product"
+                    class="card__img">
```

Comment:
Better add the same changes all over the document.
```suggestion
                <img
                  src="./images/photo-product-01.png"
                  alt="photo-product"
                  class="card__img"
                >
```